pkg/querier/blocks_store_queryable.go

```diff
@@ -614,6 +611,10 @@ func (q *blocksStoreQuerier) queryWithConsistencyCheck(
 }
 
 func newStoreConsistencyCheckFailedError(remainingBlocks []ulid.ULID) error {
+	// Sort the blocks, so it's easier to test the error strings.
+	sort.Slice(remainingBlocks, func(i, j int) bool {
+		return remainingBlocks[i].Compare(remainingBlocks[j]) < 0
+	})
```

Comment:
this was done becasue this test case fails https://github.com/grafana/mimir/blob/58e63fb6fe8b341fee2257f528dcdc4c06a405a8/pkg/querier/blocks_store_queryable_test.go#L1297

another approach to this is to have a dedicated type which holds the list of blocks and then do `assert.Contains` on that. But that complicates tests and I don't see much of a drawback of sorting